yoga/Yoga.c

```diff
@@ -94,6 +94,8 @@ typedef struct YGStyle {
   float aspectRatio;
 } YGStyle;
 
+typedef struct YGConfig { bool experimentalFeatures[YGExperimentalFeatureCount + 1]; } YGConfig;
```

Comment:
sorry, but the layouter is formating it like that, I think we need to add a second value to layout it differently. As soon as you add ```float pointScale``` this should be layouted as desired.